internal/security/secretstore/main.go

```diff
@@ -21,45 +21,43 @@ package secretstore
 
 import (
 	"context"
-	"flag"
-	"github.com/gorilla/mux"
 	"os"
 
 	"github.com/edgexfoundry/edgex-go/internal"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
 	"github.com/edgexfoundry/edgex-go/internal/security/secretstore/config"
 	"github.com/edgexfoundry/edgex-go/internal/security/secretstore/container"
 
 	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap"
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/commandline"
 	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/interfaces"
 	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/startup"
 	"github.com/edgexfoundry/go-mod-bootstrap/di"
-
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/gorilla/mux"
```

Comment:
Add empty line to separate this from go-mod-core-contracts.

Reply:
Everyone has their own preference for how the imports should be sorted. 2 blocks (standard go and the rest)  3 blocks (standard go, inner service, the rest). Now you want 4 blocks??? No where do we have this "standard" documented. 

I made the change so we can put this PR to bed, but feel this needs to be addressed. I fight with the IDE wanting just one block.

Reply:
Agree that we need to document and follow a consistent standard.